src/OrbitGl/TrackManagerTest.cpp

```diff
@@ -13,22 +15,11 @@
 
 using orbit_client_protos::TimerInfo;
 
-constexpr uint64_t kCallstackId = 1;
-constexpr uint64_t kFunctionAbsoluteAddress = 0x30;
-constexpr uint64_t kInstructionAbsoluteAddress = 0x31;
-constexpr int32_t kThreadId = 42;
-constexpr const char* kFunctionName = "example function";
-constexpr const char* kModuleName = "example module";
-constexpr const char* kThreadName = "example thread";
-
-constexpr size_t kTimerOnlyThreadId = 128;
-constexpr const char* kTimerOnlyThreadName = "timer only thread";
-
-namespace {
+namespace orbit_gl {
 
 // This is copied from CallTreeViewItemModelTest.cpp, we may want to supply this in a header,
 // but it is not clear what we want to be testing here in the future.
-std::unique_ptr<orbit_client_model::CaptureData> GenerateTestCaptureData() {
+std::unique_ptr<orbit_client_model::CaptureData> TrackTestData::GenerateTestCaptureData() {
```

Comment:
This should not be defined here, but in a file dedicated to the shared `TrackTestData`. This file corresponds to a specific test, it shouldn't also define methods that are supposed to be used across different tests (and even worse, across different modules).

Reply:
Done